scheduler_failover_controller/failover/failover_controller.py

```diff
@@ -160,25 +172,38 @@ def is_scheduler_running(self, host):
             # todo: If there's more then one scheduler running this should kill off the other schedulers. MIGHT ALREADY BE HANDLED. DOUBLE CHECK.
 
             is_running = active_list_length > 0
+            if (is_running):
+                url = "http://" + \
```

Comment:
Lets change the name of the variable to `af_health_url` or something along those lines and log the value